client/src/Component/views/PrioritiesTools/index.js

```diff
@@ -10,9 +12,25 @@ import { MyContext } from '../../../Context/ContextComponent';
 class PrioritiesTools extends Component {
   state = {};
 
+  noSelectedCardsError = () => {
```

Comment:
Hey Ali! This looks great. I will make another issue to fix the priority template beneath the alert, as well as add the alert to the click event on the methods' page next button.